packages/@okta/vuepress-site/docs/concepts/iam-overview-identity-management-factors/index.md

```diff
@@ -0,0 +1,246 @@
+
+The Universal Directory has per-group administration. You can manage external identities in UD, internal identities in AD, and mirror them in UD. You can give an application owner admin control over accounts in their domain, and give IT admin control over internal accounts. You can manage users with its REST APIs, and you can use its real-time data and reports for analysis. Its LDAP interface eliminates the need for local LDAP authentication.
+
+Using Universal Directory:
+
+- Store an unlimited number of users and attributes from sources like Active Directory, LDAP, and external IdPs, and delegate authentication to them.
+
+- Add any number of directories or domains.
+
+- Store, manage, and apply business and authentication policies that control how you handle different sets of users, including customer and partner users.
+
+- Set consistent user access policies with a central policy engine, create policies based on unique user contexts, and employ any combination of MFA factors.
+
+- Maintain user device information in device profiles that you associate with user profiles.
+
+- Define, store, and administer user roles, permissions, policies, and other data.
+
+- Organize users in hierarchical groups, and assign roles, permissions, and policies to them that are inherited by their members.
+
+- Use its out-of-the-box connections with HR systems like Workday, SaaS apps like G Suite, CSV files, and third-party identity providers.
+
+- Get a real-time syslog to troubleshoot and address security issues and use pre-built reports to see how end users use apps and services. You can download or export your data to CSV files, sync it with Security Information and Event Management systems, or access it through its API.
+
+- Admins can view and manage policies, reset passwords, and view and manage profiles of all users in their domains using its API and admin user interfaces.
+
+- Obsolete data is disabled without being deleted and can be provided to authorized admins.
+
+- You can replace your on-premise LDAP and Active Directory.
+
+Learn more:
+
+- [Universal Directory](https://www.okta.com/products/universal-directory/) lists major features and benefits of Universal Directory.
+
+### Profiles and progressive profiling
+
+Okta stores user identity information in user profiles that authorized users and admins can view, manage, and update. You can use progressive profiling to update profile data after users register.
+
+#### Okta user profiles
+
+A typical user profile contains attributes such as a user's first name, last name, username, and email address. A user is any person who is authorized to access your applications, and can be an employee, customer, partner, or other entity.
+
+Okta supports four types of user profiles:
+
+- **Okta user profiles** contain information about individual end users. They have thirty-one default base attributes, and you can add custom attributes.
```

Comment:
Perhaps not mention "thirty-one"? We may add more, then you have to remember to come in and update this....
```suggestion
- **Okta user profiles** contain information about individual end users. They have thirty-one default base attributes, and you can add custom attributes.
```